platform/web/web_main.cpp

```diff
@@ -100,6 +100,20 @@ void main_loop_callback() {
 	}
 }
 
+void print_web_header() {
+	String thread_support = OS::get_singleton()->has_feature("threads")
+			? "thread support"
+			: "single-threaded";
+	String dlink_support = OS::get_singleton()->has_feature("web_extensions")
+			? "extension support"
+			: "no extension support";
+
+	print_line(vformat(
+			"Web player features: %s, %s",
```

Comment:
I'm not sure if we should use the term "player", considering the web editor can also show this message. Maybe "web build"?